src/synth.ts

```diff
@@ -1,10 +1,27 @@
 // Classic variants are from Scale Workshop 2, they're softer on SW3.
 
 import { AperiodicWave } from 'sw-synth'
-import { centsToValue, valueToCents } from 'xen-dev-utils'
+import { centsToValue, sum, valueToCents } from 'xen-dev-utils'
 import { ceilPow2 } from './utils'
 import { computed, type ComputedRef } from 'vue'
 
+import TIMBRES from '@/timbres.json'
+
+type Spectrum = number[]; // Just its partials
+
+type Timbres = {
+  plainSpectra: { [key: string]: Spectrum }
+  // Allows migration from other data using additional keys and timbre data types
```

Comment:
This comment should be a GitHub issue or just a comment on this PR. Convert to a `// TODO: Migrate other data [...]` if it has to be here.